packages/integrations/node/test/url-protocol.test.js

```diff
@@ -2,20 +2,41 @@ import { TLSSocket } from 'tls';
 import nodejs from '../dist/index.js';
 import { loadFixture, createRequestAndResponse } from './test-utils.js';
 import { expect } from 'chai';
+import { getNetworkAddress } from '../dist/get-network-address.js'
+import { fetch } from 'undici';
 
 describe('URL protocol', () => {
 	/** @type {import('./test-utils').Fixture} */
 	let fixture;
 
 	before(async () => {
 		fixture = await loadFixture({
+			server: {
+				host: true
+		},
 			root: './fixtures/url-protocol/',
 			output: 'server',
 			adapter: nodejs({ mode: 'standalone' }),
 		});
 		await fixture.build();
 	});
 
+	describe('test preview when host is true', async () => {
+		let devPreview;
+
+		before(async () => {
+			devPreview = await fixture.preview();
+			
+		});
+		after(async () => {
+			await devPreview.stop();
+		});
+		it('test host is true ', async () => {
+			const address = getNetworkAddress('http', undefined, 3000)
+			const res = await fetch(address.network[0])
+			expect(res.status).to.equal(200);
+		});
+	})
```

Comment:
Maybe we can move this test to `node-middleware.test.js`? So we can share the preview server there so it tests faster